src/Microsoft.Data.SqlClient/tests/ManualTests/DataCommon/DataTestUtility.cs

```diff
@@ -1073,6 +1073,30 @@ public static string GetMachineFQDN(string hostname)
             return fqdn.ToString();
         }
 
+        public static bool IsIntegratedSecurity()
+        {
+            SqlConnectionStringBuilder builder = new(DataTestUtility.TCPConnectionString);
+            return builder.IntegratedSecurity;  
+        }
```

Comment:
You should be able to reference IsIntegratedSecuritySetup instead of adding this method. Then in the test, simply modify the connection string:
```
string[] removeKeys = { "Authentication", "User ID", "Password", "UID", "PWD", "Trusted_Connection" };
string connStr = DataTestUtility.RemoveKeysInConnStr(DataTestUtility.TCPConnectionString, removeKeys) +
    $"Integrated Security=true";
```